lib/tasks/erb2haml.rake

```diff
@@ -59,7 +59,7 @@ namespace :haml do
     puts '-'*80
     begin
       puts 'Would you like to delete the original .erb files? (This is not recommended unless you are under version control.) (y/n)'
-      should_delete = STDIN.gets.chomp.downcase[0]
+      should_delete = ENV["FORCE_ORIGINAL_ERB_DELETE"] ? "y" : STDIN.gets.chomp.downcase[0]
```

Comment:
This name (`FORCE_ORIGINAL_ERB_DELETE` ) looks easy to conflict for me.
But adding prefix ( such as `HAML_RAILS_`)  is too long...

Reply:
Cool, at work just now, i'll think about the variable name and add to the readme shortly.

Reply:
actually, just thinking about it, there is a use case to do the opposite too, to force to NOT delete the erb also, but still not ask the question.  I'll refactor for this too.